plugins/unicodelib/unicodelib.cpp

```diff
@@ -719,6 +720,34 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+void translate(UnicodeString & source, UChar const * sear, unsigned searLen, UChar const * repl, unsigned replLen)
+{
+    UnicodeString search(sear, searLen);
```

Comment:
Efficiency: Could be const for clarity.  Does this constructor clone the input strings?  If yes, then

     UnicodeString search(false, sear, searLen);

may be more efficient.